RELEASE_NOTES.md

```diff
@@ -84,7 +84,7 @@ This change also bubbles to the HydePage accessors, though that will only affect
 - Renamed base class AbstractPage to HydePage
 - Renamed base class AbstractMarkdownPage to BaseMarkdownPage
 - Renamed command class HydeBuildStaticSiteCommand to HydeBuildSiteCommand
-- Renamed legacy class FileCacheService to ViewDiffService
+- Renamed legacy class FileCacheService to CheckSumService
```

Comment:
Glad you caught this, I forgot I had changed it in this release. Great job!

Reply:
sure. will fix it. also please add the hacktoberfest tag to the repository has on hacktoberfest feat page it says repository not participating. Thanks

Reply:
Will do! Was actually just looking at the guidelines to make sure I'm doing everything right since this is the first Hacktoberfest PR I'm merging.

Reply:
Done, it might take 15 minutes to update. I'll be adding the tag to this PR once it's merged as well.